### PR TITLE
Allow actors to start and stop the underlying cnd instance during a test

### PR DESCRIPTION
**Pull request by thomaseizinger**
We need this extension to the framework in order to properly test #1666.

### CODE REVIEW

File: api_tests/package.json [hunk 1]
```diff
@@ -52,18 +30,39 @@
         "@types/scrypt": "^6.0.0",
```
Comment (D4nte) [inline api_tests/package.json]: Wait I am confused, did you remove the `devDependencies` section??

Comment (thomaseizinger) [reply api_tests/package.json]: Check the first commit here: https://github.com/comit-network/comit-rs/pull/1672/commits/5d3c57d360e7de605f994082a9ddbdb7438527e4

Comment (D4nte) [reply api_tests/package.json]: typo in commit: `distuingish` -> `distinguish`

**Pull request merged**